api-reference/webhook/subscription.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Event subscription"
+api: "POST https://api.travtus.com/webhook/subscribe"
+description: "Subscribe to a webhook event."
+---
+
+You can subscribe to our webhook event by calling `/webhook/subscribe` endpoint.
+We will send you the event to the endpoint url you have supplied.
+
+### Header
+
+<ParamField header="Authorization" type="string" default="none" required>
+  The authentication token for your request.
+</ParamField>
+
+### Body
+
+<ParamField body="event_name" type="string" default="none">
```

Comment:
Can you pls add a section after or before ### Body to mention what webhooks we have available? At the moment it would just be **task created**